management/actions_test.go

```diff
@@ -41,13 +41,13 @@ func TestActionManager_Create(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotEmpty(t, expectedAction.GetID())
 
-	defer cleanupAction(t, expectedAction.GetID())
+	t.Cleanup(func() {
```

Comment:
We wanna make sure we are being consistent over here and always use t.Cleanup instead of defer in our tests.